molgenis/molgenis-emx2
# feat: add variable.notes and various bugfixes found while adding

to test:
- upload molgenis.csv
- add some example data into variables, in particular 'notes'
- view results in different views

features added:
- added variables.notes to molgenis.csv (the feature request that initiated this pull request)
- added version number to molgenis.csv so we can start keeping track of changes 
- updated VariablesView to 'notes' and to have hideNA feature
- added small section to 'docs' explaning this update
- flipped default hideNA to 'true' to hide missing values so things look 'less empty' to users
-
fixes added:
- 2x upload molgenis.csv should work, which is needed in case of easy updates
- there were quite some routerlink in the breadcrumb that didn't work
- removed a {} debug in releases view
- filter input for ontology should allow multiple values
- when updating, refback trigger should be updated only for non-inherited columns
- when alter column, when key=1 (primary key) then required should be true
